src/components/profile-appearance-settings.tsx

```
@@ -0,0 +1,490 @@
+"use client";
+
+import { useEffect, useRef, useState } from "react";
+import { Button } from "@/components/ui/button";
+import { toast } from "sonner";
+import {
+    Image as ImageIcon,
```

ESLint

[error] 'ImageIcon' is defined but never used.

(unused-imports/no-unused-imports)